addons/marketing_card/models/card_campaign.py

```diff
@@ -115,6 +116,18 @@ def _get_render_fields(self):
             'content_sub_section1_path', 'content_sub_section2', 'content_sub_section2_dyn', 'content_sub_section2_path'
         ]
 
+    def _check_access_right_dynamic_template(self):
+        """Despite `_unrestricted_rendering`, a Marketing Card Manager must be able to edit the fields
+        other than the rendering fields.
+
+        The qweb rendered field `body_html` cannot be modified by users other than the `base.group_system` users,
+        as
+        - it's a related field to `card.template.body`,
+        - store=False
+        - the model `card.template` can only be altered by `base.group_system`.
+        """
```

Comment:
Suggestion a more complete explanation as the naming of the attribute and method is not crystal clear, had to dive into code to understood it again
```suggestion
        """ `_unrestricted_rendering` being True means we trust the value on model
        when rendering. This means once created, rendering is done without restriction.
        But this attribute triggers a check at create / write / translation update that
        current user is an admin or has full edition rights (group_mail_template_editor).
        
         However here a Marketing Card Manager must be able to edit the fields other
         than the rendering fields. The qweb rendered field `body_html` cannot be
         modified by users other than the `base.group_system` users, as
        - it's a related field to `card.template.body`,
        - store=False
        - the model `card.template` can only be altered by `base.group_system`
        
        Hence the security is delegated to the 'card.template' model, hence the
        check done by `_check_access_right_dynamic_template` can be bypassed.
        """
```